### ISSUE TITLE
[Rom 11:25] restoring the underlying Hebrew idiom destroyed by the Greek

### ISSUE THREAD

**Issue by Ekkles** [openenglishbible/Open-English-Bible#307]
The expression "πλήρωμα τῶν ἐθνῶν" found in the Byzantine script appears also in the Septuagint in [Gen 17:4-5; 48:19] showing this Greek to be a Greek translation of the Hebrew expression "הֲמֹון גֹּויִֽם׃" and "הֲמֹון גֹּויִם" (especially "מְלֹֽא־הַגֹּויִֽם׃" as 'multitude of nations' (H4393, H1471).

This same expression is often mis-translated as in the (ESV) 'fullness of the Gentiles' (likely because those entrusted to translate the Greek didn't read the Hebrew nearly as well - though we can compare by way of the Septuagint). This particular translation "... the whole Gentile world has been gathered in" is loaded more with theological eisegesis than linguistic acumen.

The Hebrew idiom is being lost coming to English from Hebrew, by way of the Greek which is adding error.  However, the Greek idiom once recognized as a Hebrew one can make the transition to English less violently.

The claim "πλήρωμα τῶν ἐθνῶν" is the Hebrew idiom for 'multitude of nations' from [Gen 48:19] is being made on linguistic grounds by way of the Septuagint, but it also makes sense given what Paul is saying in all of Chapter 11. His argument obtains a better coherence if it is read something like:

[Rom 11:25] "My friends, so that you don't think too highly of yourselves,  For I do not want you to be ignorant of this mystery brothers, that a partial hardening has happened to Israel, until the 'multitude of nations' has come in (or comes about)." 

Note: The clause  '.. do NOT want you to be ignorant'.. matches the Greek more than '... I want you to recognise' because of the 'οὐ' in Greek - likewise the somewhat awkward 'has come in' or 'comes about' is an effort to match the Greek (imperfect present tense) 

In other words this hardening has happened until the Abrahamic covenant was fulfilled (IAW [Gen 17:4-5][Gen 32:12][Gen 48:16,19] etc)
